solr/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved. Licensed 
```

Comment:
formatting is bad here. please try to match the other binding pom files wrt where new lines go.